cli/azd/cmd/infra_create.go

```diff
@@ -264,6 +279,13 @@ func (ica *infraCreateAction) Run(ctx context.Context, cmd *cobra.Command, args
 	}
 
 	template.CanonicalizeDeploymentOutputs(&res.Result.Properties.Outputs)
+
+	for _, svc := range proj.Services {
+		if err := svc.RaiseEvent(ctx, project.Deployed, map[string]any{"bicepOutput": res.Result.Properties.Outputs}); err != nil {
```

Comment:
I would split out the custom event arts into a separate variable on the line before for better readability